simexpal/schemes/experiments.json

```diff
@@ -56,6 +56,10 @@
 				"slurm_args": {"$ref": "#/definitions/str_or_str_list"},
 				"exclusive": {
 					"type": "boolean"
+				},
+				"max_simultaneous": {
+					"type": "integer",
+					"minimum": 1
```

Comment:
Why not `"minimum": 0`? As in: `0` would be illegal? If the key is used we have to at least provide `0`?